components/logiVehicle/fn_logiDoSpawnVehicle.sqf

```diff
@@ -100,6 +109,24 @@ if (_gearscriptType isNotEqualTo "") then
 };
 
 
+
+[_sign, _sign2]spawn
+{
+	params ["_sign", "_sign2"];
+
+
+		_dir = _sign2 getDir _sign;
+		_sign2 setDir _dir;
+
+		sleep 10;
```

Comment:
This is completely fine for now, but based on Cre8or's scheduler advice I am slowly moving the framework away from scheduled `sleep`-based timers towards CBA's `waitAndExecute` timers.  I don't want to merge new scheduled code unless the CBA equivalent code would make the logic hard to read.  If we merge it like this, I would like to create a github issue to convert this to CBA at some point.